src/localization/nav2_amcl/test/integration/test_localization_node.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) 2018 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+
+#include "amcl_node.hpp"
+#include <rclcpp/rclcpp.hpp>
+#include "std_msgs/msg/string.hpp"
+#include "geometry_msgs/msg/pose_array.hpp"
+#include "geometry_msgs/msg/pose_stamped.hpp"
+#include <memory>
+
+using std::placeholders::_1;
+
+// rclcpp::init can only be called once per process, so this needs to be a global variable
+class RclCppFixture
+{
+public:
+  RclCppFixture() { rclcpp::init(0, nullptr); }
+  ~RclCppFixture() { rclcpp::shutdown(); }
+};
+RclCppFixture g_rclcppfixture;
+
+class TestAmclPose : public ::testing::Test
+{
+public:
+  TestAmclPose()
+  {
+    //Initializing amcl_pose x and y to NaN
```

Comment:
Please run the code through cpplint and uncrustify checks. I see a few formatting issues here.

Reply:
PR #114 addresses all the ctest issues.